test/integration.js

```diff
@@ -1,8 +1,11 @@
 import Spark from '../lib/spark.js'
-import { test } from 'zinnia:test'
+
 import { assert, assertEquals } from 'zinnia:assert'
+import { test } from 'zinnia:test'
```

Comment:
I used VS Code's `Organize imports` action and I'd like to keep test dependencies separated from "lib" dependencies.